playbooks/wordpress-stack-install.yml

```diff
@@ -1,16 +1,23 @@
 ---
-- name: Install MariaDB, nginx, PHP-FPM, and WordPress
+- name: install MariaDB, nginx, PHP-FPM, and WordPress
   hosts: all
   become: yes
 
-  vars:
-    wp_db_name: wordpress-foss-magic
-    wp_db_password: 'What is a more secure way to manage passwords in a public playbook?'
-    wp_db_user: wordpress-admin
-
   roles:
     - ../roles/common-centos/7
```

Comment:
...huh. If you put the playbook in the repo root, you can just call roles by name rather than trying to give a path. Alternately, you can fiddle with `roles_path` in ansible.cfg (where you can also specify a path to your inventory file)